src/main/web/templates/handlebars/partials/cookies-banner.handlebars

```diff
@@ -5,17 +5,20 @@
         <div>
             <div class="cookies-banner__message adjust-font-size--18">
                 <h3 class="cookies-banner__heading">{{labels.cookies-banner-title}}</h3>
-                <p class="cookies-banner__body">{{{labels.cookies-banner-info}}}</p>
-                <p class="cookies-banner__body">{{labels.cookies-banner-why-we-use}}</p>
+                <p class="cookies-banner__body">{{labels.cookies-banner-info}}</p>
+                <p class="cookies-banner__body">{{{labels.cookies-banner-why-we-use}}}</p>
             </div>
             <div class="cookies-banner__buttons">
                 <div class="nojs--hide cookies-banner__button cookies-banner__button--accept">
                     <button class="btn btn--full-width btn--primary btn--focus margin-right--2 font-weight-700 font-size--17 text-wrap js-accept-cookies"
-                    data-gtm-accept-cookies="true" type="submit">{{labels.cookies-banner-accept-action}}</button>
+                    data-gtm-accept-cookies="true" type="submit" data-action="accept">{{labels.cookies-banner-accept-action}}</button>
+                </div>
+                <div class="nojs--hide cookies-banner__button cookies-banner__button--reject">
+                    <button class="btn btn--full-width btn--primary btn--focus margin-right--2 font-weight-700 font-size--17 text-wrap js-reject-cookies"
+                    data-gtm-accept-cookies="false" type="submit" data-action="reject">{{labels.cookies-banner-reject-action}}</button>
```

Comment:
1. Increase the padding between the buttons
2. Think about improving the indentation of the markup, it's gone onto multiple lines 
```html
<div>
    <button>
        {{ label }}
    </button>
</div>
```